src/last-transactions.ts

```diff
@@ -96,7 +99,8 @@ export const lastTransactions: APIGatewayProxyHandler = async (event): Promise<A
     FROM dapi_events 
     WHERE chain = $1 AND 
     event_name IN ('UpdatedBeaconWithSignedData', 'UpdatedBeaconWithPsp', 'UpdatedBeaconWithRrp', 'UpdatedBeaconSetWithBeacons', 'UpdatedBeaconSetWithSignedData') AND
-    event_data->'parsedLog'->'args'->> 0 = $2
+    event_data->'parsedLog'->'args'->> 0 = $2 AND
+    time > NOW() - INTERVAL '12 HOURS'
```

Comment:
I think 12 hours should be enough with a limit of `MAX_TRANSACTION_COUNT_LIMIT` of 720 (i.e. a max of 1 update per minute for 12 hours).

Reply:
Perfect 💪